environment.yml

```diff
@@ -19,6 +20,7 @@ dependencies:
   - raw2ometiff
   - go-mc
   - pip
+  - minio
```

Comment:
Your intent is to install minio outside of docker?

Reply:
install minio in the conda environment as per the instruction.

Reply:
> install minio in the conda environment as per the instruction.

Sorry, I still don't understand. Isn't that in the docker?

Reply:
sorry yes, I had something not correctly set up in the first instance. I started again from a clean sheet and it works nicely now